java/ql/src/semmle/code/java/Statement.qll

```diff
@@ -789,10 +791,10 @@ class LocalClassDeclStmt extends Stmt, @localclassdeclstmt {
   /** Gets the local class declared by this statement. */
   LocalClass getLocalClass() { isLocalClass(result, this) }
 
-  /** Gets a printable representation of this statement. May include more detail than `toString()`. */
   override string pp() { result = "local class declaration: " + this.getLocalClass().toString() }
 
-  /** This statement's Halstead ID (used to compute Halstead metrics). */
+  override string toString() { result = "local class declaration" }
```

Comment:
Maybe just
```suggestion
  override string toString() { result = "class ..." }
```

Reply:
Should `pp()` be changed as well? E.g. `class MyClass`

Reply:
Yeah, let's do that as well.